src/tracer/src/utils/system_info/linux_distribution.rs

```diff
@@ -0,0 +1,91 @@
+#![cfg(target_os = "linux")]
```

Comment:
I'm not sure about this file, we add this logic for what? to show a compatibility message?